src/OpenLoco/src/Ui/Window.cpp

```diff
@@ -278,7 +278,6 @@ namespace OpenLoco::Ui
             {
                 continue;
             }
-            this->callOnResize();
```

Comment:
I can see why the call to `updateViewports` was removed from `Ui.cpp`, but not why this `callOnResize` was removed. Could you elaborate?